editor/shader_create_dialog.cpp

```diff
@@ -612,6 +613,7 @@ ShaderCreateDialog::ShaderCreateDialog() {
 	// Modes.
 
 	mode_menu = memnew(OptionButton);
+	mode_menu->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
```

Comment:
The modes could be translated tbh.
But right now there is no way to expose them for translation.